jobs/JREGIONAL_PROCESS_RADARREF

```diff
@@ -76,14 +76,29 @@ gridspec_dir=${NWGES_BASEDIR}/grid_spec
 #
 #-----------------------------------------------------------------------
 #
+# create COMOUT directory
+#
+#-----------------------------------------------------------------------
+
+if [ "${RUN_ENVIR}" = "nco" ]; then 
+  if [ ${CYCLE_TYPE} == "spinup" ]; then
+    COMOUT="${COMOUT_BASEDIR}/$RUN.$PDY/${cyc}_spinup${SLASH_ENSMEM_SUBDIR}"
+  else
+    COMOUT="${COMOUT_BASEDIR}/$RUN.$PDY/${cyc}${SLASH_ENSMEM_SUBDIR}"
+  fi
+  mkdir_vrfy -p "${COMOUT}"
+fi
+#
```

Comment:
For data pre-processing, we don't need to have member directory since all the members use the same processed data. We just need to have this process and output data separate for the deterministic and ensemble run.

Reply:
removed